opendap/dtswar/src/main/java/opendap/dts/Getopts.java

```diff
@@ -0,0 +1,421 @@
+package opendap.dts;
```

Comment:
I moved a copy of this over (under `opendap/dtswar/src/main/java/opendap/util/Getopts.java `) in a [previous PR](Unidata/tds#32), but I kept the same namespace as what was in netcdf-java. I see what you did with the namespace here, though. Would you mind removing my copy of `Getopts.java` from under the `util` directory?